css/framework.css

```diff
@@ -0,0 +1,1283 @@
+}
+
+.container-m {
+  display: flex;
+  max-width: 100%;
+  flex: 1 1 100%;
+  flex-wrap: wrap;
+  padding: 0 15px;
+}
+
+.container {
+  display: flex;
+  max-width: 1300px;
+  flex: 1 1 100%;
+  flex-wrap: wrap;
+  padding: 0 15px;
+}
+
+.row {
+  display: flex;
+  max-width: 100%;
+  flex: 1 1 100%;
+  margin: 0 -15px;
+  flex-wrap: wrap;
+}
+
+.col-1-12 {
+  max-width: 8.3333333333333%;
+  flex-basis: 8.3333333333333%;
+}
+
+.col-1-12,
+.col-2-12,
+.col-3-12,
+.col-4-12,
+.col-5-12,
+.col-6-12,
+.col-7-12,
+.col-8-12,
+.col-9-12,
+.col-10-12,
+.col-11-12,
+.col-12-12 .col-1-12 {
+  display: flex;
+  flex-shrink: 1;
+}
+
+.col-2-12 {
+  max-width: 16.666666666667%;
+  flex-basis: 16.666666666667%;
+}
+
+.col-3-12 {
+  max-width: 25%;
+  flex-basis: 25%;
+}
+
+.col-4-12 {
+  max-width: 33.333333333333%;
+  flex-basis: 33.333333333333%;
+}
+
+.col-5-12 {
+  max-width: 41.666666666667%;
+  flex-basis: 41.666666666667%;
+}
+
+.col-6-12 {
+  max-width: 50%;
+  flex-basis: 50%;
+}
+
+.col-7-12 {
+  max-width: 58.333333333333%;
+  flex-basis: 58.333333333333%;
+}
+
+.col-8-12 {
+  max-width: 66.666666666666%;
+  flex-basis: 66.666666666666%;
+}
+
+.col-9-12 {
+  max-width: 75%;
+  flex-basis: 75%;
+}
+
+.col-10-12 {
+  max-width: 83.333333333333%;
+  flex-basis: 83.333333333333%;
+}
+
+.col-11-12 {
+  max-width: 91.666666666666%;
+  flex-basis: 91.666666666666%;
+}
+
+.col {
+  max-width: 100%;
+  flex-basis: 100%;
+}
+
+.col-l-1-12 {
+  max-width: 8.3333333333333%;
+  flex-basis: 8.3333333333333%;
+}
+
+.col-l-1-12,
+.col-l-2-12,
+.col-l-3-12,
+.col-l-4-12,
+.col-l-5-12,
+.col-l-6-12,
+.col-l-7-12,
+.col-l-8-12,
+.col-l-9-12,
+.col-l-10-12,
+.col-l-11-12,
+.col-l-12-12 .col-l-1-12 {
+  display: flex;
+  flex-shrink: 1;
+}
+
+.col-l-2-12 {
+  max-width: 16.666666666667%;
+  flex-basis: 16.666666666667%;
+}
+
+.col-l-3-12 {
+  max-width: 25%;
+  flex-basis: 25%;
+}
+
+.col-l-4-12 {
+  max-width: 33.333333333333%;
+  flex-basis: 33.333333333333%;
+}
+
+.col-l-5-12 {
+  max-width: 41.666666666667%;
+  flex-basis: 41.666666666667%;
+}
+
+.col-l-6-12 {
+  max-width: 50%;
+  flex-basis: 50%;
+}
+
+.col-l-7-12 {
+  max-width: 58.333333333333%;
+  flex-basis: 58.333333333333%;
+}
+
+.col-l-8-12 {
+  max-width: 66.666666666666%;
+  flex-basis: 66.666666666666%;
+}
+
+.col-l-9-12 {
+  max-width: 75%;
+  flex-basis: 75%;
+}
+
+.col-l-10-12 {
+  max-width: 83.333333333333%;
+  flex-basis: 83.333333333333%;
+}
+
+.col-l-11-12 {
+  max-width: 91.666666666666%;
+  flex-basis: 91.666666666666%;
+}
+
+.col-l-12-12 {
+  max-width: 100%;
+  flex-basis: 100%;
+}
+
+.col-m-1-12 {
+  max-width: 8.3333333333333%;
+  flex-basis: 8.3333333333333%;
+}
+
+.col-m-1-12,
+.col-m-2-12,
+.col-m-3-12,
+.col-m-4-12,
+.col-m-5-12,
+.col-m-6-12,
+.col-m-7-12,
+.col-m-8-12,
+.col-m-9-12,
+.col-m-10-12,
+.col-m-11-12,
+.col-m-12-12 .col-m-1-12 {
+  display: flex;
+  flex-shrink: 1;
+}
+
+.col-m-2-12 {
+  max-width: 16.666666666667%;
+  flex-basis: 16.666666666667%;
+}
+
```

Comment:
I don't understand clearly what you mean by classes for each break-point since there are 3, 3 col sizes are required, col-m is for medium, ie 720px-1024px screens at the bottom of the CSS there are media queries that modify each col size accordingly.